src/pages/workspaces/workspace/Data.js

```diff
@@ -1114,5 +1114,11 @@ export const navPaths = [
     path: '/workspaces/:namespace/:name/data',
     component: WorkspaceData,
     title: ({ name }) => `${name} - Data`
+  },
+  {
+    name: 'workspace-data-bucket',
+    path: '/workspaces/:namespace/:name/data/:initialDataType',
```

Comment:
Doing this felt hacky. @panentheos / @zarsky-broad any ideas on ways to do it better?

Reply:
We have an example somewhere of loading making a url including a query with something specific to load first and then removing that from the shown url once we do, I just can't remember where.

Reply:
or is it possible to pass props to the constructor directly when you call a reference link?

Reply:
Instead of using a path param, we could add an (optional) query param that the page consumes, to initialize the selected data type. That way we could remove it after the page loads so it doesn't become part of the permanent URL scheme.